bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -553,4 +573,17 @@
 		<description>Name of the scenario to trigger</description>
 	</channel-type>
 
+	<channel-type id="bypass-state">
+		<item-type>String</item-type>
```

Comment:
Perhaps this would be better represented by a `Switch` item type with values OFF, ON and UNDEF?

Reply:
Very good hint, I was not aware of the UNDEF state.